### ISSUE TITLE
Mouse tracking not reading some elements in Google Chrome unless UIA Access is enabled

### ISSUE THREAD

**Issue by ferhatbkr** [nvaccess/nvda#18707]
### Steps to reproduce:
1. Run NVDA with mouse tracking enabled.  
2. Open Google Chrome (Version 140.0.7339.16 beta, 64-bit).  
3. Move the mouse pointer over a link or certain other elements on a web page.  
4. Wait for NVDA to read the text under the mouse pointer.

### Actual behavior:
- Even though mouse tracking is enabled, links and some other elements in Google Chrome are not read by NVDA when the mouse pointer hovers over them.  
- The problem was first noticed after the latest beta update of Google Chrome, but it also occurs in NVDA versions 2024, 2025.1, and 2025.2rc1.  
- As a temporary workaround, I have to enable **“Enable UIA for Microsoft Edge and other Chromium browsers”** in NVDA’s advanced settings. However, this causes performance degradation in Chrome.

### Expected behavior:
- Links and other elements in Chrome should be read properly by mouse tracking even when UIA access is disabled.

### NVDA logs, crash dumps, and other attachments:
- 

#### NVDA installed/portable/running from source:
Installed  
#### NVDA version:
2025.2rc1 (2025.2.0.37683)  
#### Windows version:
Windows 11 Version 25H2 (OS Build 26200.5742)  
#### Name and version of other software in use when reproducing the issue:
Google Chrome Version 140.0.7339.16 beta (64-bit)  
#### Other information about your system:
Mouse tracking enabled. The issue occurs when UIA access is disabled.

#### Does the issue still occur after restarting your computer?
Yes.  
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The issue occurs in NVDA versions 2024, 2025.1, and 2025.2rc1.  
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.  
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by seanbudd** [comment]
Welcome @ferhatbkr,

We are unable to process this issue as it stands.

This seems like a duplicate [of several issues](https://github.com/nvaccess/nvda/issues?q=is%3Aissue%20state%3Aopen%20mouse%20tracking%20chrome) including:

- #8076
- #2160
- #3590
- #10920
- #3591
- #11066
- #11845
- #12047
- #14926
- #14953

Could you please specifically clarify what elements are not working for you?
If the elements you are having issues with are not covered by these tickets, please open a new issue specifying the issues you are having which are not covered by these other tickets.

As part of that, to be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

Kind Regards,
NV Access Software Developers

**Comment by hwf1324** [comment]
@seanbudd No, I don't think this is a duplicate issue. We should reopen it.

This might be an issue related to the UIA interface used in future versions of Chrome.

In my testing, when NVDA uses IA2 to find link elements via Point, it cannot find elements considered as IA2. But when NVDA switches to using UIA, it can find this link element.

CC: @jcsteh You might be able to provide a more precise explanation.

**Comment by seanbudd** [comment]
@hwf1324 - please don't tag Jamie on issues unrelated to firefox - please refer to [our experts list](https://github.com/nvaccess/nvda/blob/master/projectDocs/community/expertsList.md). 

This issue is not related to UIA, the issue reported is when UIA is not enabled.

We still need more specific information such as which elements are affected, and an HTML sample.
Without knowing the elements affected, we cannot know if it is not a duplicate of these other issues.

**Comment by hwf1324** [comment]
Currently, it is confirmed that the affected elements are link elements. As for whether there are other elements, I am not sure.

Paste the following example into the Chrome Beta address bar:

```
data:text/html;charset=utf-8,%3C!DOCTYPE%20html%3E%3Chtml%3E%3Cbody%3Ethis%20is%20%3Ca%20href%3D%22%23%22%3Ea%20link%3C%2Fa%3E%3C%2Fbody%3E%3C%2Fhtml%3E
```

**Comment by ferhatbkr** [comment]
Sorry for the delay.  

Thanks to @hwf1324  for the HTML example.  

It can also be confirmed from the example that the affected elements are links. I mostly experience this issue with links as well. I have also encountered the problem on some buttons, but I cannot confirm whether they are technically buttons or actually designed as links.  

I also reviewed the other issues mentioned. I had noticed some of them myself. However, my problem appeared with the latest beta update of Chrome. I had no issues with earlier versions. Therefore, I don’t think this is a duplicate.

**Comment by seanbudd** [comment]
Thanks for the additional information

**Comment by seanbudd** [comment]
This seems to be a regression in pre-release Chrome. Please reach out to the chromium developers to let them know about this issue, so it hopefully doesn't get released to stable Chrome.

**Comment by ferhatbkr** [comment]
Of course, I will try to reach out to the Chromium developers, but I’m not sure how to provide the necessary technical details. What is familiar to the community here may not be clear to Chromium developers.  

Also, it seems that the same issue occurs in the latest stable version of Edge as well. At least when I force UIA to be disabled, Edge is also affected by this problem. This might indicate that it has already moved beyond the early development stage.

**Comment by seanbudd** [comment]
@ferhatbkr - just provide similar information as to what you provided here. If you link the NVDA issue there, it will help too

**Comment by ferhatbkr** [comment]
I just noticed something interesting. When I move the mouse pointer over a link, NVDA sometimes reads it correctly. While this behavior feels completely random to me, it actually seems to depend on certain conditions.  

For example, when I move the mouse pointer over the link from different directions or at different speeds, this can sometimes trigger NVDA to recognize the link. It doesn’t happen every time, but in some cases, the link becomes accessible.  

Although strange, this gives me some hope that the issue might be at least partially solvable through changes on the NVDA side.

**Comment by seanbudd** [comment]
Please let us know when you've opened a Chromium issue and link the bug report here

**Comment by ferhatbkr** [comment]
I reported this issue to Chromium: [440293932](https://issues.chromium.org/issues/440293932)

**Comment by luisitochavarria-bit** [comment]
Hello, you guys!

I want to contribute to this problem.

My condition is low vision so I think my intervention could be very useful. 

This same problem is also present in the current versions of chrome and brave (the browsers in stable version) but in the stable browser version of Edge, the screen reader works perfectly.

DESCRIPTION OF THE ERROR.

within any web page, for example the WordPress administration panel when one places the mouse (mouse pointer) over the left panel, in elements that are links with text tags the reader does not read the text containing the link, nor the link itself. but on the same page if you read texts like h1, h2, p, etc. everything that is text elements reads it correctly, but if they are text tags with a link, you do not read them. 

Nor does it read the menus of a web page using the mouse pointer.

BROWSER VERSION.
Brave is up to date
Brave 1.82.161 (Official build) (64 bits)
Chromium: 140.0.7339.80

Chrome is up to date
Version 140.0.7339.81 (Official Build) (64 bits)

Microsoft Edge is updated.
Version 139.0.3405.125 (Official build) (64 bits)

**Comment by luisitochavarria-bit** [comment]
Update.

I tried one of the beta versions of the Edge browser and the problem with the reader of items with links does not read them with when you place the mouse pointer over the item. Accordingly, although the item reader with the mouse pointer works well in the current version of Microsoft Edge, it is highly likely that when upgraded, it will stop working.

Beta version of Edge:
Microsoft Edge is updated.
Version 142.0.3538.0 (Official build) canary (64 bits)

**Comment by luisitochavarria-bit** [comment]
Update.

I managed to install an older version of the Brave browser with an earlier version of Chromium, and disconnect from internet a vwz installed and delete the file that runs automatic updates.

So I was able to get the screen reader with the mouse pointer over link-type elements to work properly. 

It appears to be a problem with chromium, which will likely drag all chrome-based browsers until either the NVDA team or the chromium team resolves it.

The error occurs in chromium versions above 140.0-......

The version of the Brave browser in which the error is not present is:
Brave 1.81.137 (Official build) (64 bits)
Chromium: 139.0.7258.158

**Comment by seanbudd** [comment]
@luisitochavarria-bit - please pass this information on to the Chrome developers in the chromium bug [440293932](https://issues.chromium.org/issues/440293932)

**Comment by ryusei-48** [comment]
I've discovered something new: When you launch Chrome and open a webpage, then restart NVDA, it will temporarily start reading the content of `<a>` tags. However, if you click anywhere on the webpage or navigate to a new page, it stops reading `<a>` tags.

I can't determine whether this is a fix that needs to be implemented in NVDA or if it requires improvement on Chrome's side.

I also shared it with [440293932](https://issues.chromium.org/issues/440293932).

**Comment by seanbudd** [comment]
This has been fixed in chrome canary
